.github/workflows/push-to-main.yaml

```
@@ -0,0 +1,56 @@
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Build and bundle
+        run: |
+          cd apps/agent
+          yarn bundle
+
+      - name: Rename bundle
+        run: mv apps/agent/dist/*.tgz apps/agent/dist/agent-${{ github.sha }}.tgz
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)